common/services/component.js

```diff
@@ -19,33 +18,15 @@ function _macroNameToFilepath(macroName) {
 }
 
 function getComponent(macroName, params = {}) {
-  try {
```

Comment:
Do we want to keep this try/catch around the render, to catch any future issues?

Reply:
Hmm, this catch was to try and de-structure the params to see where the circular structure crept in. 

I don't think we necessarily _need_ to keep it long term though. Sentry will still catch the error without having to tell it to in a more specific fashion.

Reply:
:+1: